installing/installing_azure/installing-azure-user-infra.adoc

```diff
@@ -0,0 +1,127 @@
+[id="installing-azure-user-infra"]
+= Installing a cluster on Azure using ARM templates
+include::modules/common-attributes.adoc[]
+:context: installing-azure-user-infra
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+Microsoft Azure by using infrastructure that you provide.
+
+The steps for performing a user-provided infrastructure install are outlined
```

Comment:
I would remove this unnecessary sentence: The steps for performing a user-provided infrastructure install are outlined here.